config/routes.rb

```diff
@@ -7,4 +9,13 @@
   root 'certificates#index'
   get '/events', to: 'events#index'
   get '/events/:page', to: 'events#page'
+
+  get '/devauth' => 'dev_auth#show'
+  # Dashboard
+  get 'dashboard' => 'dashboard#show'
```

Comment:
Same as a above - what's the purpose of this page? could this replace the root path?